duckduckgo/Android
# Android: Extract TimeDiffFormatter / DataSizeFormatter to common module

Task/Issue URL: https://app.asana.com/0/414730916066338/1201724706502960/f

### Description
Moved `TimeDiffFormatter` and `DataSizeFormatter` to the `common` module.

### Steps to test this PR

- fresh install from this branch
- make sure App Tracking Protection is enabled

TimeDiffFormatter
- from `Settings`, go to `App Tracking Protection`
- on `App Tracking Protection` screen check `Recent Activity`
- you should see the time displayed under each entry as `Xmin ago`, `Xh ago`, `X days ago` etc

DataSizeFormatter
- on `App Tracking Protection` screen tap on the overflow menu and select `Data stats`
- on the next screen check that the values under `Data Sent` and `Data Received` are displayed in `KB`, `MB` etc

### NO UI changes
